src/Controller/Async/Stack.php

```diff
@@ -85,4 +127,22 @@ public function show(Request $request)
 
         return $this->render($template, ['context' => $context]);
     }
+
+    /**
+     * Perform, and return, a sub request for a rendered HTML stack item.
+     *
+     * @param Request $request
+     * @param string  $routeName
+     * @param string  $fileName
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    private function makeItemSubRequest(Request $request, $routeName, $fileName)
+    {
+        $panelUri = $this->generateUrl($routeName, ['fileName' => urlencode($fileName)]);
+        $subRequest = Request::create($panelUri, Request::METHOD_GET, [], $request->cookies->all(), [], $request->server->all());
+        $subRequest->setSession($request->getSession());
+
+        return $this->app->handle($subRequest, HttpKernelInterface::SUB_REQUEST, false);
+    }
```

Comment:
Eww I don't like that at all. I would just go back to rendering with twig directly.